x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/page.tsx

```diff
@@ -145,10 +145,11 @@ export const Page: FC = () => {
       <SavedObjectsWarning onCloseFlyout={refresh} />
       <UpgradeWarning />
 
-      {mapJobId || mapModelId || analyticsId ? (
+      {jobId || modelId ? (
```

Comment:
it looks like these could all be `??`

Reply:
Good call - updated in 0f91240b80423f8b7b1843165a71525be92eaea5